aspnetcore/fundamentals/logging/index.md

```diff
@@ -893,9 +629,253 @@ Using a third-party framework is similar to using one of the built-in providers:
 
 For more information, see each provider's documentation. Third-party logging providers aren't supported by Microsoft.
 
+<a name="nhca"></a>
+
+## Non-host console app
+
+For an example of how to use the Generic Host in a non-web console app, see the *Program.cs* file of the [Background Tasks sample app](https://github.com/dotnet/AspNetCore.Docs/tree/master/aspnetcore/fundamentals/host/hosted-services/samples) (<xref:fundamentals/host/hosted-services>).
+
+Logging code for apps without Generic Host differs in the way [providers are added](#add-providers) and [loggers are created](#create-logs). 
+
+### Non-host console app Logging providers
```

Comment:
I realise there might be a good reason for the prefix here, so feel free to ignore (well, that's always the case 😄).

```suggestion
### Logging providers
```